databases/sql-server/README.md

```diff
@@ -0,0 +1,191 @@
+# Microsoft SQL Server example
+
+This example shows how to use Prisma to introspect an SQL Server database and use [Prisma Client](https://www.prisma.io/docs/reference/tools-and-interfaces/prisma-client) in a **Node.js script** to read and write data in an SQL Server database. You can find the database schema in [`./schema.sql`](./schema.sql).
+
+The example consists of two parts:
+
+- `tests/prisma.test.ts`: Jest test (in TypeScript) with a variety of Prisma Client queries and assertions to showcase access patterns
+- `src/script.js`: Node.js script with queries similar to the ones in the test.
+
+## How to use
+
+### 1. Download example & install dependencies
+
+Clone this repository:
+
+```
+git clone git@github.com:prisma/prisma-examples.git --depth=1
+```
+
+Install npm dependencies:
+
+```
+cd prisma-examples/databases/sql-server
+npm install
+```
+
+### 2. Start SQL Server with Docker Compose
+
+Run the following command from the `sql-server` folder to start SQL Server:
+
+```
+docker-compose up -d
+```
+
+> **Note:** The `docker-compose.yml` is where the super admin password is set with the `SA_PASSWORD` environment variable
+
+### 3. Create the database in SQL Server with `schema.sql`
+
+The `schema.sql` file contains the SQL queries to create the database schema (which includes the database and tables).
+
+Run `schema.sql` with the following [SQL Server `sqlcmd` CLI](https://docs.microsoft.com/en-us/sql/tools/sqlcmd-utility?view=sql-server-ver15) command:
+
+```
+sqlcmd -S 127.0.0.1 -U SA -P prisma123 -i schema.sql
+```
+
+Alternatively, you can use one of the following tools to create the database schema:
+
+- [Azure Data Studio](https://docs.microsoft.com/en-us/sql/azure-data-studio/what-is?view=sql-server-ver15)
+- [TablePlus](https://tableplus.com/)
+- [DBeaver](https://dbeaver.io)
+
+### 4. Configure the database connection URL
+
+Prisma will use the `DATABASE_URL` environment variable in `.env` to connect to the database.
+Create the file:
+
+```
+touch .env
+```
+
+Then add the following line:
+
+```
+DATABASE_URL=sqlserver://localhost:1433;database=prisma;user=SA;password=prisma123;trustServerCertificate=true;encrypt=true
+```
+
+> **Note:** The database URL uses the `SA` (super admin) user of the database and the same password as defined in `docker-compose.yml`.
+> If you are running on macOS, you may need to use `encrypt=DANGER_PLAINTEXT` to work around the current TLS limitation.
+
+### 5. Introspect your database
+
+The Prisma schema is the foundation for the generated Prisma Client API. Therefore, you first need to make sure that all the database tables are represented in it. The easiest way to do so is by introspecting your database:
+
+```
+npx prisma introspect
+```
+
```

Comment:
```suggestion

If the command times out, append `encrypt=DANGER_PLAINTEXT` to your connection URL in `.env` so that it looks as follows: `DATABASE_URL=sqlserver://localhost:1433;database=prisma;user=SA;password=prisma123;trustServerCertificate=true;encrypt=true;encrypt=DANGER_PLAINTEXT`

```

I ran into the timeout and didn't make the connection to the previous note. I think it would be helpful to be more explicit about this issue here again so that people know what to do.